src/barplot.ts

```diff
@@ -69,41 +78,61 @@ export class BarPlot extends AVisInstance implements IVisInstance {
     return <Element>this.$node.node();
   }
 
+  transform(scale?: [number, number], rotate: number = 0) {
+    const bak = {
+      scale: this.options.scale || [1, 1],
+      rotate: this.options.rotate || 0
```

Comment:
It is defined in IVisInstanceOptions - I haven't touched that. IBarPlotOptions just extends IVisInstanceOptions.